modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -165,7 +165,7 @@ def buildProject(Map options = [:]) {
     lock("$repoName-$NODE_NAME-test") {
       if (hasDatabase()) {
         stage("Set up the database") {
-            runRakeTask("db:drop db:create db:schema:load")
+            runRakeTask("db:drop db:create db:schema:load_if_ruby db:structure:load_if_sql")
```

Comment:
One thought I just had is that when you run the tests locally, you don't run `rake db:seed` before you run the tests, you just run the tests.  I'm not 100% convinced that having CI run the seeds automatically before running tests is a good thing in this case because it's different to what a dev would be doing when running the tests.  If we wanted the seeds to act like fixtures in the test dbs, it'd be worth making that explicit in each app that needs it so that everyone makes sure it's run.  What rails will run before each test run (if invoked via rake, not necessarily via rspec) is `rake db:test:prepare` which checks for missing migrations and then runs `db:schema:load` (or `db:structure:load` as appropriate).